packages/sanity/src/core/error/sentry/sentryErrorReporter.ts

```diff
@@ -101,7 +101,7 @@ export function getSentryErrorReporter(): ErrorReporter {
     if (hasThirdPartySentry) {
       client = new BrowserClient({
         ...clientOptions,
-        transport: makeFetchTransport,
+        transport: makeBufferedTransport,
```

Comment:
Should initialize the transport in the below `init` call as well, I think? Potentially move it to `clientOptions` if possible?

Reply:
ah, apologies -- victim of a hasty refactor. Should be sorted now.